### PR TITLE
Add RHEL7 SELinux support for new service_name_v6 param, subsequently fix puppet lint error

### PR DESCRIPTION
**Pull request by wilson208**
This originally started as a fix for a puppet lint issue in CI, where the $seluser variable had been removed from the firewall::linux::redhat class as seluser property for the File["/etc/sysconfig/${service_name}"] resource is now set in a case statement. 

When a separate PR was merged adding the 'service_name_v6' param, it still used the $seluser variable to set the seluser property of the File["/etc/sysconfig/${service_name_v6}"] resource.

When both these PR's were merged they then caused a puppet lint error in CI.

### CODE REVIEW

**Comment by ferventcoder** [comment]
I'd prefer to see the additional fields be added as a separate commit than the lint fixes.

**Comment by wilson208** [comment]
@ferventcoder I began with solving the lint issue. However I noticed the lint issue was because the $seluser variable had been removed and the result of solving the lint issue is this PR. I didn't add the additional service_name_v6 param, it was added in PR #641, and the variable $seluser was removed in PR #658. The PR for 641 was created in June and not merged until last week, which is why this was missed when david made his changes in #658 at the end of november.

File: manifests/linux/redhat.pp [hunk 1]
```diff
@@ -85,23 +92,28 @@
         case $::operatingsystemrelease {
           /^7\..*/: {
             case $::operatingsystem {
-              'CentOS': { File["/etc/sysconfig/${service_name}"] { seluser => 'unconfined_u', seltype => 'system_conf_t' } }
-              default : { File["/etc/sysconfig/${service_name}"] { seluser => 'unconfined_u', seltype => 'etc_t' } }
+              'CentOS': {
+                File["/etc/sysconfig/${service_name}"] { seluser => 'unconfined_u', seltype => 'system_conf_t' }
+                File["/etc/sysconfig/${service_name_v6}"] { seluser => 'unconfined_u', seltype => 'system_conf_t' }
```
Comment (ferventcoder) [inline manifests/linux/redhat.pp]: This looks like it could be moved to its own commit

**Comment by ferventcoder** [comment]
Now I better understand what is going on after looking closer.

**Comment by ferventcoder** [comment]
Minor nit* - would love to see the git commit message summary be around 50 chars with a message body. I looked over history for this repo and didn't find any long summaries.
